Bottom: 9bb3990ea89770c9805d5c5eaac8f4726423c9e7
Top:    93f770df5a9f0c11495cfcc90d56f1a809b13721
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 14:47:51 -0500

Refresh of zarlink

---

diff --git a/.gitignore b/.gitignore
index 0392deb..bceaf90 100644
--- a/.gitignore
+++ b/.gitignore
@@ -22,6 +22,6 @@
 *#
 .emacs.desktop*
 TAGS*~
-Packages
-Packages.gz
-patches*
+/Packages
+/Packages.gz
+/patches*
